doc/source/releases/23.0_announce_user.rst

```diff
@@ -74,12 +74,17 @@ It's easier than ever to change the datatype of every dataset in a collection us
    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/SNvne6GdKzs" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen alt="Video of a user using the new search interface to find collection operations tools, and then a Filter tool."></iframe>
 
 
-RO-Crate / Biocompute Export
+RO-Crate / Biocompute Workflow Invocation Export
 ----------------------------
 
-Are you interested in publishing workflow artefacts? You can do that in one of two formats now, RO-Crate and BioCompute Object. These are both standards based efforts to package research data and metadata. Read more about `RO-Crate <https://www.researchobject.org/ro-crate/>`__ and `BioCompute Objects <https://biocomputeobject.org/>`__. You'll need to have a remote file source configured (e.g. FTP or Dropbox) to use it.
+Galaxy now offers two options for packaging and publishing workflow artifacts: RO-Crate and BioCompute Object. These standard-based formats provide a way to bundle research data and metadata in a structured way. To learn more about these formats, check out RO-Crate <https://www.researchobject.org/ro-crate/>__ and BioCompute Objects <https://biocomputeobject.org/>__. Please note that you'll need to enable the Celery task system in order to export to both formats. Additionally, you can directly export these files to any writeable remote file source that has been configured (e.g., FTP or Dropbox).
 
 
+History Export Tracking
+----------------------------
+
+Good news for users with Celery enabled on their Galaxy instances: a new History export user interface is now available with additional options for generating temporal History downloads. This feature also allows you to package your history as an RO-Crate and export it to remote file sources for more permanent storage. Plus, every time you export your history to one of these remote sources, a new tracking record is created with a button that lets you re-import that history snapshot with ease.
```

Comment:
Could you add a screenshot here?

```suggestion
A new History export user interface is now available that makes it easier to download your histories. You can export your history in a variety of formats, and either download it directly, or save it to a remote file source for more permanent storage. Plus, every time you export your history to one of these remote sources, a new tracking record is created with a button that lets you re-import that history snapshot with ease.
```

again I think we can just say this without confusing users over what celery is or how they could enable it. I think we should pair this with a note in the admin facing release notes that mentions Celery is required for these new features that your users might be asking about.

Reply:
You're right, I was just a bit anxious about users being confused because these features are not visible in big instances like main.

Reply:
We'll have celery on main pretty soon, I think it's not worth stressing about this.